peas/weather.py

```diff
@@ -784,175 +740,55 @@ def calculate_and_set_PWM(self):
                 ))
                 self.set_PWM(new_PWM)
 
-    def make_safety_decision(self, current_values):
-        """
-        Method makes decision whether conditions are safe or unsafe.
-        """
-        self.logger.debug('Making safety decision')
-        self.logger.debug('Found {} weather data entries in last {:.0f} minutes'.format(
-            len(self.weather_entries), self.safety_delay))
-
-        safe = False
-
-        # Tuple with condition,safety
-        cloud = self._get_cloud_safety(current_values)
-
-        try:
-            wind, gust = self._get_wind_safety(current_values)
-        except Exception as e:
-            self.logger.warning('Problem getting wind safety: {}'.format(e))
-            wind = ['N/A']
-            gust = ['N/A']
-
-        rain = self._get_rain_safety(current_values)
-
-        safe = cloud[1] & wind[1] & gust[1] & rain[1]
-        self.logger.debug('Weather Safe: {}'.format(safe))
-
-        return {'Safe': safe,
-                'Sky': cloud[0],
-                'Wind': wind[0],
-                'Gust': gust[0],
-                'Rain': rain[0]}
-
     def _get_cloud_safety(self, current_values):
```

Comment:
Modify to use results from `_get_status()`

Reply:
Modified to use the results from `_get_status()`.